### PR TITLE
yori: Change to directly downloading and extracting

### PR DESCRIPTION
**Pull request by niheaven**

### CODE REVIEW

**Comment by rashil2000** [comment]
This can be updated now as https://github.com/ScoopInstaller/Scoop/pull/3518 has been released

File: bucket/yori.json [hunk 1]
```diff
@@ -3,11 +3,41 @@
...
+            "hash": [
+                "54fff9392774bb29685503626d4f8a2610c07fdc36ac13ac67bc44f7ccc9b0fd",
+                "065d485d621362f98419857a58648373c7c283b320c0237d37cf082d5096cf35",
+                "e72fa905b3f75eaccb4bf67a891cddfa1e2f5a6ff1398f5e435d2aeffac4bbb8",
+                "73350e264d145ed8fc22bb885f096cae2d7de28e5060b76bc2e79cd23abe8792",
+                "8a6c75796d1f482f07c4b73f5243d7e3e52c67ca5b3b07ee62376abcfd82011f"
+            ]
+        }
     },
+    "post_install": "Remove-Item \"$dir\\pkginfo.ini\" -Force",
```
Comment (rashil2000) [inline bucket/yori.json:40]: Why delete this?

Comment (niheaven) [reply bucket/yori.json:40]: There is pkginfo.ini in every cab package which contains package info, so it is unused after installation.

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**